clients/web/src/lib/components/atoms/form/Input/Input.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts" context="module">
+    export type InputSize = "sm" | "md" | "lg";
+</script>
+
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import shortid from "shortid";
+
+    import type {FormControlState} from "../form.types";
+
+    export let size: InputSize = "md";
+    export let label: string;
+    export let placeholder: string | undefined = undefined;
+    export let disabled: boolean = false;
+    export let state: FormControlState = "none";
+    export let error: string | undefined = undefined;
+
+    const labelId = `input_${shortid.generate()}`;
+    const errorId = `input-error_${shortid.generate()}`;
+</script>
+
+<div class="size-{size} state-{state}">
```

Comment:
If size and state are missing; do we want to continue attaching these classes?

Reply:
Size and state have default values of `md` and `none` respectively, so they won't be missing, and we do want to apply the classes